code/game/objects/items/devices/scanners.dm

```diff
@@ -448,36 +448,41 @@ BREATH ANALYZER
 	throw_range = 7
 	matter = list(DEFAULT_WALL_MATERIAL = 30,"glass" = 20)
 
-/obj/item/device/slime_scanner/attack(mob/living/M as mob, mob/living/user as mob)
-	if (!isslime(M))
-		to_chat(user, "<B>This device can only scan slimes!</B>")
+/obj/item/device/slime_scanner/attack(mob/living/M, mob/living/user)
+	if(!isslime(M))
+		to_chat(user, span("warning", "This device can only scan slimes!"))
 		return
 	var/mob/living/carbon/slime/T = M
-	user.show_message("Slime scan results:")
-	user.show_message(text("[T.colour] [] slime", T.is_adult ? "adult" : "baby"))
-	user.show_message(text("Nutrition: [T.nutrition]/[]", T.get_max_nutrition()))
-	if (T.nutrition < T.get_starve_nutrition())
-		user.show_message("<span class='alert'>Warning: slime is starving!</span>")
+	to_chat(user, span("notice", "**************************"))
+	to_chat(user, span("notice", "Slime scan results:"))
+	to_chat(user, span("notice", capitalize_first_letters("[T.colour] [T.is_adult ? "adult" : "baby"] slime")))
+	to_chat(user, span("notice", "Health: [T.health]"))
+	to_chat(user, span("notice", "Nutrition: [T.nutrition]/[T.get_max_nutrition()]"))
+	if(T.nutrition < T.get_starve_nutrition())
+		to_chat(user, span("alert", "Warning: slime is starving!"))
 	else if (T.nutrition < T.get_hunger_nutrition())
-		user.show_message("<span class='warning'>Warning: slime is hungry</span>")
-	user.show_message("Electric change strength: [T.powerlevel]")
-	user.show_message("Health: [T.health]")
-	if (T.slime_mutation[4] == T.colour)
-		user.show_message("This slime does not evolve any further")
+		to_chat(user, span("warning", "Warning: slime is hungry!"))
+	to_chat(user, span("notice", "Electric charge strength: [T.powerlevel]"))
+	to_chat(user, span("notice", "Growth progress: [T.amount_grown]/10"))
+	if(T.cores > 1)
+		to_chat(user, span("warning", "Anomalous number of slime cores detected."))
+	else if(!T.cores)
+		to_chat(user, span("warning", "No slime cores detected."))
+	to_chat(user, span("notice", "Genetic Information:"))
+	if(T.slime_mutation[4] == T.colour)
+		to_chat(user, span("warning", "This slime cannot evolve any further."))
 	else
-		if (T.slime_mutation[3] == T.slime_mutation[4])
-			if (T.slime_mutation[2] == T.slime_mutation[1])
-				user.show_message(text("Possible mutation: []", T.slime_mutation[3]))
-				user.show_message("Genetic destability: [T.mutation_chance/2]% chance of mutation on splitting")
+		if(T.slime_mutation[3] == T.slime_mutation[4])
+			if(T.slime_mutation[2] == T.slime_mutation[1])
+				to_chat(user, span("notice", "Possible mutation: [T.slime_mutation[3]]"))
```

Comment:
Shouldn't this be listing two mutations? 1 and 3?

Reply:
I genuinely have no idea, old code had it like this and no one's made an issue about it yet, so I assume it's working as intended.

Reply:
I think this is incorrect, but I'll fix it in a future PR of my own I guess.